play-scala/caching-in-play/build.sbt

```diff
@@ -0,0 +1,22 @@
+name := """caching-in-play"""
+organization := "com.baeldung"
+
+version := "1.0-SNAPSHOT"
+
+lazy val root = (project in file(".")).enablePlugins(PlayScala)
+
+scalaVersion := "2.13.3"
+
+libraryDependencies += guice
+libraryDependencies += caffeine
+libraryDependencies += ws
+libraryDependencies += "org.scalatestplus.play" %% "scalatestplus-play" % "5.0.0" % Test
+libraryDependencies += "org.mockito" % "mockito-core" % "3.5.13" % Test
+
+PlayKeys.devSettings += "play.server.http.port" -> "9000"
+
+// Adds additional packages into Twirl
```

Comment:
Let's remove these lines that are commented out.

Reply:
Done.